src/edu/washington/escience/myria/coordinator/MasterCatalog.java

```diff
@@ -113,6 +114,7 @@
     + "    col_index INTEGER NOT NULL,\n"
     + "    col_name TEXT,\n"
     + "    col_type TEXT NOT NULL,\n"
+    + "    is_indexed INTEGER NOT NULL, \n" 
```

Comment:
I can't think of specific cases where this might be useful. I forgot about lack of support for ORDER BY. In the past, creating indexes was useful when pushing down selection operations to Postgres. For example, with the cosmo data, I would need to find particles that belonged to a particular galaxy (grpId). For that scenario, it was useful to have an index for the grpID column, but if I were to run that today, I would also need enable  pushing down the computation to postgres. 

Long story short, I don't think there is a strong case for index ordering. It seems like indexes will be useful only for selections that are pushed down to postgres....